src/browser/components/EditionView.jsx

```diff
@@ -30,7 +30,7 @@ export const EnterpriseOnlyFrame = ({ command }) => {
       </p>
       <p>
         Find out more over at{' '}
-        <a href="https://neo4j.com/editions/" target="_blank">
+        <a href="https://neo4j.com/editions/" target="_blank" rel="noreferrer">
```

Comment:
Eslint now complains about "_blank" links without norefferrer because of [this vurn](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)